docs/source/manual/configuration.rst

```diff
@@ -767,6 +767,94 @@ Name                   Default      Description
 type                   REQUIRED     The filter type.
 ====================== ===========  ================
 
+.. _man-configuration-json-layout:
+
+JSON layout
+------------
+
+.. code-block:: yaml
+
+    layout:
+      type: json
+      timestampFormat: "yyyy-MM-dd'T'HH:mm:ss.SSSZ"
+      includeTimestamp: true
+      prettyPrint: false
+      appendLineSeparator: true
+      includeLevel: true
+      includeThreadName: true
+      includeMDC: true
+      includeLoggerName: true
+      includeFormattedMessage: true
+      includeMessage: false
+      includeException: true
+      includeContextName: false
+      customFieldNames:
+        timestamp: "@timestamp"
+
+=======================  ===========  ================
+Name                     Default      Description
+=======================  ===========  ================
+includeTimestamp         true         Whether to include the timestamp of the event to the JSON map.
```

Comment:
Here and below, I think the phrasing should be "include x _in_ the JSON map" rather than "include x to the JSON map".